app/src/main/java/vn/mbm/phimp/me/newGallery.java

```diff
@@ -3698,7 +3729,7 @@ public boolean onLongClick(View v) {
 		ImageView imageSelector;
 	}
 
-	public static void resumeLocalPhoto(){
+	public void resumeLocalPhoto(){
```

Comment:
I don't think so.  everything is working fine. Actually that part was changed to resolve merge conflict at that time.